### ISSUE TITLE
Migrating to Cordova@9.0.0

### ISSUE THREAD

**Issue by rstylesoftlab** [infobip/mobile-messaging-cordova-plugin#18]
Hello.
We are trying to migrate from cordova@8.1.2 to cordova@9.0.0. 
We use your plugin version 1.1.5.

We have error adding a platfrorm on cordova@9.0.0:
`Using "requireCordovaModule" to load non-cordova module "q" is not supported. Instead, add this module to your dependencies and use regular "require" to load it.
`
As we understand, using requireCordovaModule() should be replaced with require().

Other plugins released new versions to support this breaking change of cordova@9.0.0.
For example cordova-custom-config released version 5.1.0:
https://github.com/dpa99c/cordova-custom-config/blob/master/CHANGELOG.md

Could you provide support of cordova@9.0.0?

**Comment by rstylesoftlab** [comment]
Could you answer something please?

**Comment by tjuric** [comment]
Hi @rstylesoftlab we intended to take this issue the first week of October.

**Comment by kas-kad** [comment]
Hello @rstylesoftlab. I could not reproduce your issue.
```
```
```
$ cordova plugin add com-infobip-plugins-mobilemessaging
Adding com-infobip-plugins-mobilemessaging to package.json
+ cordova-custom-config@5.1.0
added 37 packages in 5.137s
```

**Comment by kas-kad** [comment]
Hello @rstylesoftlab, we've released version 1.2.0 that supports Cordova 9.

**Comment by rstylesoftlab** [comment]
It is great! Thank you.
